### ISSUE TITLE
maven repository for BIRT

### ISSUE THREAD

**Issue by albertgao228** [eclipse/birt#1158]
I am trying to upgrade birt runtime for my application from 4.4 to 4.10. But I have hard time to find the maven package available anywhere. Can you please consider to make birt run time package available in public maven repo?

Thanks!

**Comment by m-thirumal** [comment]
Currently, we are stuck with `4.4.2`
`<dependency>
		    <groupId>org.eclipse.birt.runtime</groupId>
		    <artifactId>org.eclipse.birt.runtime</artifactId>
		    <version>4.4.2</version>
		</dependency>`

**Comment by BR-Nicholas-Prado** [comment]
Duplicates #625.

In the meantime, it's worth noting that - if you trust innovent solutions - you can use [birt 4.8.0](https://repo1.maven.org/maven2/com/innoventsolutions/birt/runtime/org.eclipse.core.runtime_3.14.0.v20180417-0825/4.8.0/). You may be able to hack together a set of dependencies for 4.9 from what has been published recently.

**Comment by albertgao228** [comment]
I tried birt 4.8.0 packaged under innoventsolutions. The issues I experienced were related that it packaged a lot of basic dependencies with older version. To be specific - slf4j - after upgrading it to 4.8.0, I got compilation error saying "java.lang.String cannot be converted to org.slf4j.Marker".
I tried the solution to exclude certain package path - but it didn't work well with multi-module projects.
